securedrop_client/locale/fr/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,372 @@
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n > 1;\n"
+"X-Generator: Weblate 4.14.1\n"
+"Generated-By: Babel 2.10.3\n"
+
+msgid "{application_name} is already running"
+msgstr "{application_name} est déjà lancé"
+
+msgid "Retrieving 1 message"
+msgid_plural "Retrieving {message_count} messages"
+msgstr[0] "Récupération d’un message"
+msgstr[1] "Récupération de {message_count} messages"
+
+msgid "The SecureDrop server cannot be reached. Trying to reconnect..."
+msgstr "Impossible d’atteindre le serveur SecureDrop. Tentative de reconnexion…"
+
+msgid ""
+"Could not reach the SecureDrop server. Please check your \n"
+"Internet and Tor connection and try again."
+msgstr ""
+"Impossible d’atteindre le serveur SecureDrop. Veuillez vérifier vos\n"
+"connexions à Internet et à Tor et réessayez."
+
+msgid ""
+"Those credentials didn't work. Please try again, and \n"
+"make sure to use a new two-factor code."
+msgstr ""
+"Ces authentifiants sont erronés. Veuillez réessayer et\n"
+"utiliser un nouveau code d’authentification à deux facteurs."
+
+msgid "That didn't work. Please check everything and try again."
+msgstr "Cela n’a pas fonctionné. Veuillez tout vérifier et réessayer."
+
+msgid "You must sign in to perform this action."
+msgstr "Vous devez vous connecter pour effectuer cette action."
+
+msgid "Your session expired. Please log in again."
+msgstr "Votre session a expiré. Veuillez vous reconnecter."
+
+msgid "Failed to update star."
+msgstr "Échec de mise à jour de l’étoile."
+
+msgid "File does not exist in the data directory. Please try re-downloading."
+msgstr "Le fichier n’existe pas dans le répertoire des données. Veuillez tenter de retélécharger."
+
+msgid "The file download failed. Please try again."
+msgstr "Échec de téléchargement. Veuillez réessayer."
+
+msgid "Failed to delete conversation at server"
+msgstr "Échec de suppression de la conversation sur le serveur"
+
+msgid "Failed to delete source at server"
+msgstr "Échec de suppression de la source sur le serveur"
+
+msgid "No messages."
+msgstr "Aucun message."
+
+#, python-format
+msgid "%(sender)s wrote:"
+msgstr "%(sender)s a écrit :"
+
+#, python-format
+msgid "%(sender)s sent:"
+msgstr "%(sender)s a envoyé :"
+
+#, python-format
+msgid "File: %(filename)s"
+msgstr "Fichier : %(filename)s"
+
+msgid "Download All"
+msgstr ""
+
+msgid "Delete Source Account"
+msgstr "Supprimer le compte de la source"
+
+msgid "Delete All Files and Messages"
+msgstr "Supprimer tout les fichiers et messages"
+
+msgid "Print Transcript"
+msgstr ""
+
+msgid "Export Transcript"
+msgstr ""
+
+msgid "Export All"
+msgstr ""
+
+msgid "<h2>Some files will not be exported</h2>Some files from this source have not yet been downloaded, and will not be exported.<br /><br />To export the currently-downloaded files, click \"Continue.\""
+msgstr ""
+
+msgid "all files and transcript"
+msgstr ""
```

Comment:
:thinking: I don't know where this string is coming from…

Reply:
Looks like:

https://github.com/freedomofpress/securedrop-client/blob/aa6d07d5d2a06c27844095d635c1885e256604aa/securedrop_client/gui/actions.py#L340